pulsar-flink-connector/src/main/java/org/apache/flink/streaming/connectors/pulsar/internal/ReaderThread.java

```diff
@@ -142,31 +140,26 @@ protected void createActualReader() throws PulsarClientException {
         reader = readerBuilder.create();
     }
 
-    protected void handleUnAvailedCursor() throws PulsarClientException {
+    protected void handleUnAvailedStartMessageId() throws PulsarClientException {
```

Comment:
what's the meaning of `UnAvailed`?

Reply:
@nlu90 StartMessageId may have an invalid value. In this case, it should be handled according to the rules. Do you have any better suggestions for this method name?

Reply:
@nlu90 I have modified it, do you think it is feasible?